server/src/validation/items.validation.ts

```diff
@@ -58,4 +58,41 @@ export const validateAddItemRequest = () => {
                 throw new Error('categoryId does not exist');
             })
     ];
-};
\ No newline at end of file
+};
+
+export const validateEditItemRequest = () => {
+    return [
+        body('_id', 'Invalid itemId'),
```

Comment:
What does the `_id` validation do? I don't think it does anything.
I believe you might want to do `...validateDeleteItemRequest()` instead.
You can rename everything so it'll be more logical.